### ISSUE TITLE
Question about 2D simulations

### ISSUE THREAD

**Issue by hongshul** [InteractiveComputerGraphics/SPlisHSPlasH#29]
Hi,

I run the 2D simulations and find strange phenomena.

When I choose PBF/IISPH/DFSPH/ProjectiveFluids as  the simulation mehtod, the energy of the fluid is too sufficient. DFSPH even creates vortices.

When I choose PCISPH as the simulation mehtod, the fluid will explode.

Do you see these phenomenon?

Cheers

**Comment by janbender** [comment]
Probably you tested the dam break 2D example. In this one I added the micropolar material model to add vortices. So that is a feature not a bug ;-) If you turn off the micropolar vorticity model, the vortices should not appear. 

PCISPH is less stable than the other methods. We investigated this in more detail in our DFSPH paper if you want to know more about this. PCISPH therefore requires a lower time step size. Either take a smaller fixed step size or use a smaller CFL factor to solver this.

In general it is hard to find a parameter set that works for all solvers without decreasing the performance of the more stable methods.
